### PR TITLE
RS: 7.4.2 Fuji June maintenance release docs

### PR DESCRIPTION
**Pull request by rrelledge**
DOC-3483

### CODE REVIEW

File: content/operate/rs/release-notes/rs-7-4-2-releases/_index.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ For more detailed release notes, select a build version from the following table
 
 ## Version changes
 
+### Product lifecycle updates
+
+Redis Enterprise Software version 6.2 is approaching end-of-life, scheduled for August 31, 2024. After this date, version 6.2 will not be included in [supported upgrade paths]({{<relref "/operate/rs/installing-upgrading/upgrading/upgrade-cluster#supported-upgrade-paths">}}) for future major releases of Redis Enterprise Software.
```
Comment (mich-elle-luna) [inline content/operate/rs/release-notes/rs-7-4-2-releases/_index.md:51]:
```suggestion
Redis Enterprise Software version 6.2 is approaching end-of-life, scheduled for August 31, 2024. After this date, version 6.2 will not be included in [supported upgrade paths]({{<relref "/operate/rs/installing-upgrading/upgrading/upgrade-cluster#supported-upgrade-paths">}}) for major releases of Redis Enterprise Software.
```

File: content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+- RS118231: Fixed output of `rladmin tune db` when updating buffer limits fails due to unreachable shards.
+
+- RS128091: `XREADGROUP` on CRDB should properly update idle time.
+
+## Version changes
+
+### Product lifecycle updates
+
+Redis Enterprise Software version 6.2 is approaching end-of-life, scheduled for August 31, 2024. After this date, version 6.2 will not be included in [supported upgrade paths]({{<relref "/operate/rs/installing-upgrading/upgrading/upgrade-cluster#supported-upgrade-paths">}}) for future major releases of Redis Enterprise Software.
```
Comment (mich-elle-luna) [inline content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md:80]:
```suggestion
Redis Enterprise Software version 6.2 is approaching end-of-life, scheduled for August 31, 2024. After this date, version 6.2 will not be included in [supported upgrade paths]({{<relref "/operate/rs/installing-upgrading/upgrading/upgrade-cluster#supported-upgrade-paths">}}) for major releases of Redis Enterprise Software.
```

File: content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+- (CVE-2023-41056) In some cases, Redis may incorrectly handle resizing of memory buffers, which can result in incorrect accounting of buffer sizes and lead to heap overflow and potential remote code execution.
+
+- (CVE-2023-41053) Redis does not correctly identify keys accessed by `SORT_RO` and, as a result, may grant users executing this command access to keys that are not explicitly authorized by the ACL configuration. (Redis 7.0.13)
+
+- (CVE-2023-36824) Extracting key names from a command and a list of arguments may, in some cases, trigger a heap overflow and result in reading random heap memory, heap corruption, and potentially remote code execution. Specifically: using `COMMAND GETKEYS*` and validation of key names in ACL rules. (Redis 7.0.12)
+
+- (CVE-2023-28856) Authenticated users can use the `HINCRBYFLOAT` command to create an invalid hash field that will crash Redis on access. (Redis 7.0.11)
+
+- (CVE-2023-28425) Specially crafted `MSETNX` command can lead to assertion and denial-of-service. (Redis 7.0.10)
```
Comment (mich-elle-luna) [inline content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md:155]:
```suggestion
- (CVE-2023-28425) A specially crafted `MSETNX` command can lead to assertion and denial-of-service. (Redis 7.0.10)
```

**Review by mich-elle-luna — COMMENTED** [review]
Thank you! Just suggest to remove the word 'future' for clarity and one typo.

File: content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+- [RediSearch 2.8.13]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisearch/redisearch-2.8-release-notes.md" >}})
+
+- [RedisJSON 2.6.10]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisjson/redisjson-2.6-release-notes.md" >}})
+
+- [RedisTimeSeries 1.10.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redistimeseries/redistimeseries-1.10-release-notes.md" >}})
+
+- [RedisBloom 2.6.12]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisbloom/redisbloom-2.6-release-notes.md" >}})
+
+- [RedisGears 2.0.20](https://github.com/RedisGears/RedisGears/releases/tag/v2.0.20-m21)
```
Comment (LiorKogan) [inline content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md:44]: Please change "RedisGears 2.0.20" to  "RedisGears 2.0.20 (preview)"

File: content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+- `region_name` field for new AWS S3 regions
+
+- Bug fixes
+
+## New in this release
+
+### Enhancements
+
+- Added the `region_name` field in the REST API to import, export, and back up data in new AWS S3 regions.
+
```
Comment (LiorKogan) [inline content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md:29]: Why do we add release notes for the modules and not for "Redis database" itself?
(we have it only in GitHub I believe - e.g., https://github.com/redis/redis/releases/tag/7.2.5).

**Comment by adisht** [comment]
Hi @rrelledge Since we have both Redis Enterprise Software 6.2.* and Redis database 6.2, I think it would be better to state the full "Redis Enterprise Software 6.2.*" in the upgrade note, otherwise it may be confusing.
Specifically note that Redis database 6.2 is bundled with the next Redis Enterprise Software (7.6.*), and it will allow DB upgrades from Redis DB 6.2 -> Redis DB 7.4, but the enterprise product will not allow upgrading from a Redis Enterprise Software 6.2.* -> Redis Enterprise Software 7.6.* (the future numbers should not specified yet)
I hope it is clear :) Thanks!

File: content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+- [RedisGraph v2.10.15]({{< relref "/operate/oss_and_stack/stack-with-enterprise/release-notes/redisgraph/redisgraph-2.10-release-notes.md" >}})
+
+### Resolved issues
+
+- RS124753: Fixed a bug with databases created using a recovery plan with replication enabled where shards did not write persistence files.
+
+- RS126290: Allow editing an existing Replica Of source's proxy certificate in the Cluster Manager UI.
+
+- RS126476: Fixed unexpected connection errors in the DMC proxy log.
```
Comment (yoavredis) [inline content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md:64]: @rrelledge "database-audit" should be added, i.e. it should be:

RS126476: Fixed unexpected database-audit connection errors in the DMC proxy log.

**Review by yoavredis — DISMISSED** [review]
@rrelledge LGTM except for 1 comment I've added regarding resolved issue RS126476.

File: content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md (hunk 4 above)
Comment (rrelledge) [reply content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md]: Fixed

**Review by mich-elle-luna — APPROVED** [review]
thank you!

**Pull request merged**